docs/upgrading.md

```diff
@@ -0,0 +1,30 @@
+# Upgrading
+
+This page lists the breaking API changes between major Kinto.js version, as well as upgrade guides.
+
+## 1.x to 2.x
+
+Kinto.js 2.x introduces transactions, and the Adapter base interface has been updated accordingly:
```

Comment:
I would introduce this part with the comment you put at the end «*This is internal adapter stuff*».